Makefile

```diff
@@ -140,9 +147,11 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/crd | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
-.PHONY: deploy
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+_setImage: kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+
+.PHONY: deploy
+deploy: manifests kustomize _setImage ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
could we use `tilt ci` to do this instead of manually deploying things here or is it worth testing this outside of tilt as well?

Reply:
@eljohnson92 We have two different environments. Tilt for development and Kuttl for testing. It doesn't make much sense to introduce cross project complexity to avoid a make target. And this method has been generated by Kubebuilder so ~all Kubebuilder projects have this function. Anyway I'm open to other solutions, my focus was KIS. It is possible to call Tilt from Kuttl to deploy the system before executing the tests. Or it is also possible to start the environment via Tilt and use Kuttl only to execute the tests themselves (we can start Kuttl from Tilt). So there are options. WDYT?

Reply:
given this is for test I don't want to block review on this, my preference would be we use Tilt to call `tilt ci` instead of `make deploy` so we can just use the existing deployment interface. if you have a strong opinion that we should leave it as it is for now I would be ok with leaving this as it and giving it a thumbs up. let me know what you think